http/src/client/mod.rs

```diff
@@ -1275,6 +1275,93 @@ impl Client {
         UpdateRolePositions::new(self, guild_id, roles)
     }
 
+    /// Create a new guild based on a template.
+    ///
+    /// This endpoint can only be used by bots in less than 10 guilds.
+    ///
+    /// # Errors
+    ///
+    /// Returns [`CreateGuildFromTemplateError::NameInvalid`] when the name is
+    /// invalid.
+    ///
+    /// [`CreateGuildFromTemplateError::NameInvalid`]: crate::request::template::create_guild_from_template::CreateGuildFromTemplateError::NameInvalid
+    pub fn create_guild_from_template(
+        &self,
+        template_code: impl Into<String>,
+        name: impl Into<String>,
+    ) -> StdResult<CreateGuildFromTemplate<'_>, CreateGuildFromTemplateError> {
+        CreateGuildFromTemplate::new(self, template_code, name)
+    }
+
+    /// Create a template from the current state of the guild.
+    ///
+    /// Requires the `MANAGE_GUILD` permission. The name must be at least 1 and
+    /// at most 100 characters in length.
+    ///
+    /// # Errors
+    ///
+    /// Returns [`CreateTemplateError::NameInvalid`] when the name is invalid.
+    ///
+    /// Returns [`CreateTemplateError::DescriptionTooLarge`] when the
```

Comment:
This error can't result from this call, only separately on `CreateTemplate::description`, so this should be removed from the list.